[33,23]
error: 'name' has private access in 'Cat1'